kouncil-frontend/migrations.json

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
remove this file

Reply:
has been already removed. Check latest commit :)